onnxruntime/core/platform/posix/env.cc

```diff
@@ -278,10 +278,12 @@ class PosixEnv : public Env {
 
   // Return the number of physical cores
   int GetNumPhysicalCpuCores() const override {
+#if !defined(__APPLE__) && !defined(__ANDROID__) && !defined(__wasm__) && !defined(_AIX)
 #ifdef CPUINFO_SUPPORTED
```

Comment:
Same comments apply to all similar occasions in this file - can we merge the ifdefs in consecutive lines.